samples/Config.Options.Snapshot.Web/Config.Options.Snapshot.Web.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Curious what happens during restore though? Is Mono required on the Linux/MacOS when users try to restore?